feature/experimental/tunnel/otg_tests/tunnel_interface_based_resize_test/README.md

```diff
@@ -0,0 +1,90 @@
+ *   Configure an IPv4 and an IPv6 address for each of the 32xtunnel interfaces.
+ *   Add 32 different v4 and v6 static routes for the same /24 IPv4 and /64 IPv6 destination prefixes. These static routes should point at the 32 different tunnel interface addresses by their respective address families as next-hop.
+ *   Add static routes for the /27 IPv4 and /64 IPv6 tunnel destination prefixes to point at lag1, lag2 and port6 for ECMP.
+ *   The test should send 1000 IPv4 and 1000 1Pv6 flows from the ATE:Port1 to the DUT:Port1. The flows should have variations in the following,
+        * Source IP address
+        * Source Port
+        * Destination IP adresses. These will be from the same /24 and /64 destination prefixes for which the static routes were added
+        * Destination Port
+        * Payload protocol
+ *   Incoming traffic on DUT:PORT1 should be load shared across all 32 Tunnel interfaces for encapsulation and then routed out the combination of lag and singleton interfaces, i.e. lag1, lag2 and Port6. Follow all the verifications steps for Sub-test1 below
+ *   Reduce the number of Tunnel interfaces(e.g. From 32 to 16):
+        * If the static routes are used to forward traffic to tunnel, please disable or delete the static route in this test to simulate the reduction in available paths
+* Incoming traffic on DUT:PORT1 should be load shared across all remainging 16 Tunnel interfaces. Follow all the verifications steps for Sub-test1 below
+* Increase number of Tunnel interfaces(e.g. From 16 to 32)
+* Incoming traffic on DUT:PORT1 should now be load shared across all 32 Tunnel interfaces. Follow all the verifications steps for Sub-test1 below
+* Increasing and decreasing the number of tunnel interfaces and related static route shouldn’t case traffic drops.
+        
+
+## SUB-Test:2 Encap Test
+
+## Verification steps for Subtest-1
+*   Verify the next hop counters for packet being diverted or sent for encapsulation
+        *   Verify the tunnel interfaces counters to confirm traffic encapsulation
+        *   Verify the tunnel interfaces for optimal load balancing of traffic/flow. Acceptable deviation is <=3%
+
+## Verification steps for Subtest-2
+ *   After decapsulation, traffic should be load balanced/hashed to all available L3 ECMP which untimately resolves over a combination of LAG and Singleton interfaces. Acceptable deviation is <=3%.
+ *   Verify the tunnel interfaces counters to confirm the traffic decapsulation
+ *   Validate system for:
+        * Health-1.1
+        * No feature related error or drop counters incrementing
+
+## Verification steps for Subtest-2    
+*   After decapsulation, traffic should be load balanced/hashed to all available L3 ECMP or LAG or combination of both features
```

Comment:
Decap doesn't apply here.

Reply:
Right... doesnt makes sense now that I look at it. Removed in the updated version of the test